liota/dcc/vrops.py

```diff
@@ -155,26 +156,25 @@ def publish(self, metric):
         self.con.send(message)
 
     def init_relations(self, gw):
-      """ This function initializes all relations between gateway and it's children.
-          It is called after each object's UUID is received.
+        """ This function initializes all relations between gateway and it's children.
```

Comment:
It was intentional. Because originally it was 2 spaces per level - only for these two methods, which was not consistent which any part of the same file or any file in the repo.